CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 ### Changed
 ### Fixed
+- Fix blank screen on initial login from [meain](https://github.com/meain)
```

Comment:
Dropping adding the `in [#00](https://github.com)` section at the end as we will only know the PR number after a PR is created plus I guess  you can always run blame on the file and figure it out more or less.